src/domain/collection/curatedEvents/__tests__/CuratedEvents.test.tsx

```diff
@@ -159,16 +149,41 @@ test('should show expired events', async () => {
 });
 
 test('event list pagination works', async () => {
+  await paginationTest({ eventType: 'event' });
+});
+
+test('course list pagination works', async () => {
+  setFeatureFlags({ EVENTS_HELSINKI_2: true });
```

Comment:
This will left the feature flag on for the rest of the test run? 
Perhaps all the tests in the file could be wrapped into `describe` which has the
```
afterEach(() => {
  jest.restoreAllMocks();
});
```

Reply:
There is this in the top of the file
```
afterEach(() => {
  jest.restoreAllMocks();
});
```